engine/getstarted/last_page.md

```diff
@@ -9,56 +9,46 @@ title: Learn more
 notoc: true
 ---
 
-This getting started provided very basic essentials for using Docker on Mac, Windows, and Linux. If you want to learn more with regard to end-to-end development, start with the full install instructions and feature overviews, then follow up with more advanced tutorials and user guides.
+This getting started provided very basic essentials for using Docker. If you want to learn more with regard to end-to-end development, start with the full install instructions and feature overviews in the platform-specific guides, then follow up with more advanced tutorials and user guides.
```

Comment:
Maybe "This topic provided"? Or "This Getting Started guide"? Maybe s/with regard to/about